src/prpl.ts

```diff
@@ -194,6 +194,12 @@ self.addEventListener('activate', () => self.registration.unregister());`);
   };
 }
 
+/**
+ * Promisified version of `fs.exists`.
+ */
+function fileExists(filepath: string): Promise<boolean> {
+  return new Promise((resolve) => fs.exists(filepath, resolve));
```

Comment:
`fs.exists` is deprecated. See https://nodejs.org/api/fs.html#fs_fs_exists_path_callback.

Maybe also use `util.promisify()` instead. https://nodejs.org/api/util.html#util_util_promisify_original

Reply:
Done, switched to `fs.stat`. Don't want to use `util.promisify` quite yet because it's only supported in Node 8, and right now I'm testing on 6 and latest (along with all the other tools team projects). 8 actually went to LTS today, so maybe soon we can bump everything up.